doc/source/quickstart.rst

```diff
@@ -127,11 +127,30 @@ To access the loaded data use the wavelength or name:
 
     >>> print(global_scene[0.6])
 
-To visualize loaded data in a pop-up window:
-
-    >>> global_scene.show(0.6)
-
-To make combine datasets and make a new dataset:
+Visualizing data                                                                                    
+================                                                                                    
+
+To visualize loaded data in a pop-up window:                                                        
+                                                                                                    
+    >>> global_scene.show(0.6)                                                                      
+                                                                                                    
+Alternatively if working in a Jupyter notebook the scene can be converted to a `geoviews            
+<http://geo.holoviews.org/index.html>` object using the :meth:`~satpy.scene.Scene.to_geoviews`      
+method.                                                                                             
```

Comment:
Could you make it a little clearer in this paragraph that `geoviews` (and any other packages) must be installed for this to work?

Reply:
yes